content/en/docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -1,184 +1,232 @@
 ---
 reviewers:
-- aveshagarwal
-- eparis
-- thockin
-title: Fedora (Single Node)
+- chrisnegus
+title: Kubernetes on Fedora (Single Node)
 ---
 
-{{< toc >}}
+* TOC
+{:toc}
+
+To try out Kubernetes on Fedora, you have a few choices:
+
+* **kubeadm** - The kubeadm command provides a toolkit to bootstrap a Kubernetes cluster that is reasonable secure, extensible, and upgradable. See this document for details on using kubeadm with flannel networking to set up Kubernetes in Fedora (either as a single-node or multi-node configuration).
```

Comment:
> kubeadm is a tookit for bootstrapping a Kubernetes cluster that is reasonably secure, extensible and upgradable.

Reply:
Is this paragraph missing a link? I'd expect "See this document" to link somewhere